### PR TITLE
MAINT: Raise minimum to 3.10, bump actions

### PR DESCRIPTION
**Pull request by mgxd**

### CODE REVIEW

File: nibabies/cli/workflow.py [hunk 1]
```diff
@@ -132,14 +130,16 @@ def build_boilerplate(workflow):
         from shutil import copyfile
         from subprocess import CalledProcessError, TimeoutExpired, check_call
 
-        from pkg_resources import resource_filename as pkgrf
+        from nibabies.data import load as load_data
+
+        bib = load_data.readable("boilerplate.bib").read_text()
```
Comment (effigies) [inline nibabies/cli/workflow.py:135]:
```suggestion
        bib = load_data("boilerplate.bib")
```

File: nibabies/conftest.py [hunk 2]
```diff
@@ -33,5 +35,5 @@ def data_dir():
 @pytest.fixture(autouse=True)
 def set_namespace(doctest_namespace, data_dir):
     doctest_namespace["data_dir"] = data_dir
-    doctest_namespace["test_data"] = Path(resource_filename("nibabies", "tests/data"))
+    doctest_namespace["test_data"] = load_data.as_path('.')
```
Comment (effigies) [inline nibabies/conftest.py:38]:
```suggestion
    doctest_namespace["test_data"] = load_data()
```

Comment (mgxd) [reply nibabies/conftest.py]: the intended path is actually `nibabies.tests.data`, it seems to be happy with `load_data.cached('../tests/data')`

Comment (effigies) [reply nibabies/conftest.py]: Fair enough. I generally just instantiate a new loader in `tests/data/__init__`.

**Pull request merged**